### ISSUE TITLE
React Docs - Clicking header scrolls too far

### ISSUE THREAD

**Issue by ahtee** [reactjs/reactjs.org#1727]
im reading the react docs to learn ( 😃) and clicked on the header title and noticed the page scrolls a bit too far.

[https://reactjs.org/docs/introducing-jsx.html#jsx-is-an-expression-too](https://reactjs.org/docs/introducing-jsx.html#jsx-is-an-expression-too)

**Comment by ahtee** [comment]
This usually happens when a navbar has an absolute position, from past experience.

**Comment by alexkrolick** [comment]
> This usually happens when a navbar has an absolute position, from past experience.

Yep, it's using the browser's built-in handling of #ids that have been added to headings, so the top bar overlaps. PR welcome to fix this.
